js/components/closed-caption-multi-audio-menu/closedCaptionMultiAudioMenu.js

```diff
@@ -12,7 +13,7 @@ var ClosedCaptionMultiAudioMenu = React.createClass({
    * @returns {Array<{id: String, label: String, enabled: Boolean}>} an array of languages info objects
    * @private
    */
-  getClosedCaptions: function(languageList, language) {
+  getClosedCaptions: function (languageList, language) {
```

Comment:
This is completely subjective, but I prefer not having spaces between `function` and parenthesis. Objectively, I think most of our codebase doesn't use spaces though.

Reply:
That's a great point!
![image](https://user-images.githubusercontent.com/19288322/38347023-4ce52562-38a1-11e8-8c30-66f73ad533ac.png)
![image](https://user-images.githubusercontent.com/19288322/38347032-5e26e9aa-38a1-11e8-91fe-4fea459ad9c4.png)
I'll update the PR and ESLint